apps/i18n/common/fr_fr.json

```diff
@@ -1,16 +1,19 @@
 {
   "abuseDetail": "Merci de donner des précisions au sujet du contenu que tu signales",
   "abuseFormDetail": "Merci de fournir autant de détails que possible au sujet du contenu que tu signales.",
+  "abuseFormAcknowledge": "En soumettant ces informations, vous reconnaissez qu'elles seront traitées conformément aux termes de la [Politique de Confidentialité] ({privacy_url}) et des [Conditions d'Utilisation] ({tos_url}).",
```

Comment:
@castro-jorge I'm seeing a lot of issues with Markdown formatting and message format formatting (see 67bb975). I'm fixing the message format formatting because it blocks the sync but it looks like a lot of these translations are from TBH.

Reply:
Alrighty, will look into their translations